### PR TITLE
feat(kube): add autoscaling to node pools

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-6681
| License          | BSD 3-Clause

## Description

Allow customers to scale their k8s cluster.
Reverts commit 4f14978c

### CODE REVIEW

**Comment by mohammed-zahaf** [comment]
Hello @antleblanc and @marie-j, i reverted this feature.
I completed it by adding the required fix.

**Review by radireddy — APPROVED** [review]

**Review by varun257 — DISMISSED** [review]

**Comment by marie-j** [comment]
Hello @mohammed-zahaf 
I notice that the reference in the PR description doesn't match the one in the commit, which one should be taken into account ? 
Thanks

**Comment by mohammed-zahaf** [comment]
Hello@marie-j, I updated the description.
It must be MANAGER-6681.

Thanks.

**Review by marie-j — DISMISSED** [review]

**Comment by marie-j** [comment]
@mohammed-zahaf could you please solve existing conflicts ? 
Thanks

**Comment by mohammed-zahaf** [comment]
Hello @JeremyDec and @marie-j.
I juste rebased this branch on master.

**Review by radireddy — APPROVED** [review]

**Review by radireddy — APPROVED** [review]

**Comment by mohammed-zahaf** [comment]
Hello @marie-j and @JeremyDec.
The reporter has made and validate QA on master branch.

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/translations/Messages_fr_CA.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/translations/Messages_fr_CA.json:1]: No translation for other languages ?

**Review by JeremyDec — CHANGES_REQUESTED** [review]
Hello @mohammed-zahaf, I see only `fr_CA` / `fr_FR` translations files for `packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/translations/Messages_fr_FR.json`.

Is there missing translations ?

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/translations/Messages_fr_CA.json (hunk 1 above)
Comment (JeremyDec) [reply packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/translations/Messages_fr_CA.json:1]: Ok, other translations already exists in `master`

**Comment by JeremyDec** [comment]
Seen together, other files already exists in `master`

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**